README.md

```diff
@@ -16,10 +16,12 @@ The auditing focuses on the entire SDLC process, where it can reveal risks from
 [![Downloads][download]][release]
 [![Build Status](https://github.com/aquasecurity/chain-bench/workflows/Build/badge.svg?branch=main)](https://github.com/aquasecurity/chain-bench/actions)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/aquasecurity/chain-bench/blob/main/LICENSE)
+[![go-report-card][go-report-card]](https://goreportcard.com/report/github.com/aquasecurity/chain-bench)
 
 [download]: https://img.shields.io/github/downloads/aquasecurity/chain-bench/total?logo=github
 [release-img]: https://img.shields.io/github/release/aquasecurity/chain-bench.svg?logo=github
 [release]: https://github.com/aquasecurity/chain-bench/releases
+[go-report-card]: https://goreportcard.com/badge/github.com/aquasecurity/chain-bench?style=flat-square
```

Comment:
```suggestion
[download]: https://img.shields.io/github/downloads/aquasecurity/chain-bench/total?logo=github&style=flat-square
[release-img]: https://img.shields.io/github/release/aquasecurity/chain-bench.svg?logo=github&style=flat-square
[release]: https://github.com/aquasecurity/chain-bench/releases
[go-report-card]: https://goreportcard.com/badge/github.com/aquasecurity/chain-bench?style=flat-square
```

![](https://goreportcard.com/badge/github.com/aquasecurity/chain-bench?style=flat-square)
is different to existing badges, make them all flat square

![](https://img.shields.io/github/downloads/aquasecurity/chain-bench/total?logo=github) vs ![](https://img.shields.io/github/downloads/aquasecurity/chain-bench/total?logo=github&style=flat-square)